src/univalue/univalue.cpp

```diff
@@ -164,6 +164,19 @@ int UniValue::findKey(const std::string& key) const
     return -1;
 }
 
+const UniValue UniValue::findValue(const std::string& key) const
+{
+    if (typ != VOBJ)
+        return NullUniValue;
```

Comment:
This is the only real non-trivial change,  it now defines the behaviour more specifically for non-`VOBJ`.
Wouldn't this have worked already as `keys` would be empty?